homeassistant/components/fritzbox/__init__.py

```diff
@@ -143,6 +143,11 @@ def __init__(
         self._device_class = entity_info[ATTR_DEVICE_CLASS]
         self._attr_state_class = entity_info[ATTR_STATE_CLASS]
 
+    @property
+    def available(self) -> bool:
+        """Return if entity is available."""
+        return self.coordinator.last_update_success and self.device.present
```

Comment:
Upstream class implements the available method already 

```suggestion
        return super().available and self.device.present
```